apiserver/observer/request_notifier.go

```diff
@@ -95,7 +94,7 @@ func (n *RequestObserver) Login(entity names.Tag, model names.ModelTag, fromCont
 		if !n.state.fromController {
 			n.connLogger.Infof("agent login: %s for %s", n.state.tag, n.state.model)
 		}
-		n.hub.Publish(apiserver.ConnectTopic, apiserver.APIConnection{
+		_, _ = n.hub.Publish(apiserver.ConnectTopic, apiserver.APIConnection{
```

Comment:
This seems dangerous that we're dropping the error return on the floor unilaterally.
Thank you for catching it, but it should be something we follow up on quickly. Even if the answer is just logging that we got an error.